platform/mellanox/mlnx-ssd-fw-update.sh

```diff
@@ -178,6 +179,10 @@ function check_usage() {
             ARG_POWER_CYCLE_FLAG=$TRUE
             shift # past argument
             ;;
+	--no-power-cycle)
```

Comment:
@alexrallen this will break `Innodisk` FW update. Please sync with BSP for more details

Reply:
All this does is suppress the *automatic* power cycle. With the way this is currently configured the auto update flow will still power cycle the machine immediately after installation (perhaps after finishing installing another firmware). 

The power cycle performed here is the same one performed by the register by the platform specific reboot plugin.

Reply:
The user is given a message to power cycle the box immediately if this is called from the firmware install flow.